src/components/filterModal/statsList/StatsList.vue

```diff
@@ -193,7 +193,8 @@ export default {
         inheritanceHandler(stat) {
             if (stat.title === 'Inheritance') {
                 return !stat.data.filter(item => !this.hasProblemGroup(item) &&
-                    this.filteredData(item).variants).length || !stat.data.length;
+                    (stat.type !== STAT_TYPE_ZYGOSITY || this.filteredData(item).variants)
+                    ).length || !stat.data.length;
```

Comment:
Is there a sense to check stat.data.length after filtering? Additionally check it by eslint, there are 5 errors.